views-react/build.gradle

```diff
@@ -0,0 +1,86 @@
+plugins {
+    id "io.micronaut.build.internal.views-module"
+    id "com.github.node-gradle.node" version "7.0.2"
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
+    api projects.micronautViewsCore
+    api libs.jetbrains.annotations
```

Comment:
don't depend on jet brains annotations, we have our own `Nullable` `NonNull` if you need those